docs/leo-rover/manuals/frame-and-suspension.mdx

```diff
@@ -1,643 +1,1040 @@
+            alt="3x M4x10 torx screw"
+            sources={{
+              light: useBaseUrl('/img/robots/leo/manuals/frame-V2/2-4-black.webp'),
+              dark: useBaseUrl('/img/robots/leo/manuals/frame-V2/2-4-white.webp'),
+            }}
+            width="2300"
+            height="2400"
+            caption="3x M4x10 button head screw"
+          />
+  </FlexTableItem>
+</FlexTable>
+
+<ThemedImageZoom
+  alt="Step 1 - 1"
+  sources={{
+    light: useBaseUrl('/img/robots/leo/manuals/frame-V2/2-black.webp'),
+    dark: useBaseUrl('/img/robots/leo/manuals/frame-V2/2-white.webp'),
+  }}
+  width="1480"
+  height="2200"
+  figStyle={{
+    width: 600,
+  }}
+/>
+
+<ThemedImageZoom
+  alt="Step 1 - 2"
+  sources={{
+    light: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-black.webp'),
+    dark: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-white.webp'),
+  }}
+  width="1480"
+  height="2200"
+  figStyle={{
+    width: 600,
+  }}
 />
 
-<ImageZoom
-  src="/img/robots/leo/manuals/frame-suspension/frame-suspension-64.webp"
-  width="1200"
-  height="802"
-  caption="Slide the beam until the scratch lines up with the differential arm edge.
-      Keep the parts together."
-/>
+---
 
-<ImageZoom
-  src="/img/robots/leo/manuals/frame-suspension/frame-suspension-65.webp"
-  width="1200"
-  height="802"
-  caption="Tighten the screws using a 4mm hex key."
+## Step 2
+
+<FlexTable style={{ alignContent: 'center', justifyContent: 'space-evenly'}}>
+  
+  <FlexTableItem
+    style={{
+      width: '40%'
+    }}
+    mobileColumns={2}>
+      <ThemedImageZoom
+            alt="Step 1 assembly"
+            sources={{
+              light: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-1-black.webp'),
+              dark: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-1-white.webp'),
+            }}
+            width="2300"
+            height="2400"
+          />
+  </FlexTableItem>
+
+<FlexTableItem
+  style={{
+    textAlign: 'center',
+    width: '40%',
+  }}
+  mobileColumns={2}
+>
+  <ThemedImageZoom
+    alt="Bearing support"
+    sources={{
+      light: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-4-black.webp'),
+      dark: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-4-white.webp'),
+    }}
+    width="2300"
+    height="2400"
+  />
+</FlexTableItem>
+
+    <FlexTableItem
+    style={{
+      textAlign: 'center',
+      width: '40%'
+    }}mobileColumns={2}>
+      <ThemedImageZoom
+            alt="8x M4 nut"
+            sources={{
+              light: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-3-black.webp'),
+              dark: useBaseUrl('/img/robots/leo/manuals/frame-V2/3-3-white.webp'),
+            }}
+            width="2300"
+            height="2400"
+            caption="8x M4 nut"
+          />
+
+  </FlexTableItem>
+
+   <FlexTableItem
+    style={{
+      textAlign: 'center',
+      width: '40%'
+    }}mobileColumns={2}>
+      <ThemedImageZoom
+            alt="8x M4x20 cylinder head screw"
```

Comment:
In previous documents I saw these screws refered as allen screws. Please unify spelling. Maybe use ISO standard?

Reply:
Unified.